drivers/filesystems/fs_rec/ext.c

```diff
@@ -19,21 +19,21 @@
 
 BOOLEAN
 NTAPI
-FsRecIsExt2Volume(IN PEXT2_SUPER_BLOCK SuperBlock)
+FsRecIsExtVolume(IN PEXT_SUPER_BLOCK SuperBlock)
 {
     /* Just check for magic */
-    return (SuperBlock->Magic == EXT2_SUPER_MAGIC);
+    return (SuperBlock->Magic == EXT_SUPER_MAGIC);
 }
 
 NTSTATUS
 NTAPI
-FsRecExt2FsControl(IN PDEVICE_OBJECT DeviceObject,
+FsRecExtFsControl(IN PDEVICE_OBJECT DeviceObject,
                    IN PIRP Irp)
```

Comment:
I understand this would increase the diff, but I would suggest to fix the indentation of these 2nd+ parameters

Reply:
done